README.md

```diff
@@ -57,7 +57,7 @@ Stage 2 indicates that the committee expects these features to be developed and
 | [Function implementation hiding][censorship]                                   | Domenic Denicola<br />Michael Ficarra                 | Michael Ficarra                                                                   |                                            | <sub>[June&nbsp;2020][censorship-notes]</sub>                         |
 | [collection normalization][collection-rekey]                                   | Bradley Farias                                        | Bradley Farias                                                                    |                                            | <sub>[January&nbsp;2019][richer-keys-notes]</sub>                     |
 | [isTemplateObject][isTemplateObject]                                           | Mike Samuel, Krzysztof Kotowicz                       | Daniel Ehrenberg<br />Jordan Harband                                              |                                            | <sub>[April&nbsp;2024][isTemplateObject-notes]</sub>                  |
-| [`Map.prototype.emplace`][map-emplace]                                         | Bradley Farias                                        | Erica Pramer                                                                      |                                            | <sub>[July&nbsp;2020][map-emplace-notes]</sub>                        |
+| [`Upsert`][upsert]                                                             | Daniel Minor, Lauritz Thoresen Angeltveit, Jonas Haukenes, Sune Lianes, Vetle Larsen, Mathias Hop Ness | Daniel Minor | Michael Ficarra, Jesse Alama | <sub>[December&nbsp;2024][upsert-notes]</sub>                        |
```

Comment:
we only have two columns for names, but there's three here.

Reply:
The third one is for reviewers :)

Reply:
aha, i forgot we just added that :-p my bad